test/presubmit-backend-test.sh

```diff
@@ -14,14 +14,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# usage: `./hack/run_unit_tests_backend.sh` to run backend unit tests once
-#        `WATCH=true ./hack/run_unit_tests_backend.sh` to watch code changes and auto rerun tests
-# Note: ibazel can be downloaded from https://github.com/bazelbuild/bazel-watcher
-
-COMMAND="bazel"
-if [ -n "$WATCH" ]; then
-  COMMAND="ibazel"
-fi
-$COMMAND --host_jvm_args=-Xmx500m --host_jvm_args=-Xms500m test \
-  --noshow_progress --noshow_loading_progress --define=grpc_no_ares=true \
-  --test_output=all //backend/...
+# The current directory is /home/prow/go/src/github.com/kubeflow/pipelines
+# 1. install go in /home/prow/go1.13.3
```

Comment:
Instead of installing go here, I think we should update the image we use to run backend test:
https://github.com/kubernetes/test-infra/blob/c16594a1a99e0e11c1e042368e6a925a311b5e92/config/jobs/kubeflow/kubeflow-presubmits.yaml#L346

because we also want to run this script when testing locally.